ensime-client.el

```diff
@@ -995,6 +995,14 @@ copies. All other objects are used unchanged. List must not contain cycles."
      ,max-results
      )))
 
+(defun ensime-rpc-public-symbol-search
```

Comment:
we probably need to do this for all the API calls, abstract the API from the implementation because it **will** change.